### ISSUE TITLE
Feature Request: If table is included in perspective, any subsequent calc cols/measures be automatically included.

### ISSUE THREAD

**Issue by seedjay1** [TabularEditor/TabularEditor3#342]
That is, new calculated columns/measures on a table should "inherit" the perspective(s) of the containing table, by default.

I only just now realized this isn't already the case, and am having to go back through the model to find the "gaps". :(

edit: Alt means of achieving same goal: have some kind of "apply to all children" check box when assigning a table to a perspective.

**Comment by tcardoni** [comment]
in Preferences > Modeiling Operations > Perspective Membership Inheritance, the options work to inherit the perspective from the table when manually adding a new column or measure.  I would like to see it work when choosing Update table schema as well

**Comment by tcardoni** [comment]
![image](https://user-images.githubusercontent.com/39279710/151042065-5fac58ba-df46-4877-be85-b71d53d61e85.png)

**Comment by otykier** [comment]
Hi @tcardoni 
Thanks for posting this. We will make sure this works when columns are added through the Update Table Schema feature as well!

In the mean time, you can use this C# script to align perspective membership of columns, measures and hierarchies to that of the parent table:

```csharp
foreach(var table in Selected.Tables)
{
    foreach(var measure in table.Measures) measure.InPerspective.CopyFrom(table.InPerspective);
    foreach(var column in table.Columns) column.InPerspective.CopyFrom(table.InPerspective);
    foreach(var hierarchy in table.Hierarchies) hierarchy.InPerspective.CopyFrom(table.InPerspective);
}
```

**Comment by seedjay1** [comment]
Thank you both!

**Comment by otykier** [comment]
This is now fixed in [3.2.1](https://github.com/TabularEditor/TabularEditor3/releases/tag/3.2.1). Thanks for reporting!
